### DISCUSSION TITLE
Online Sync Across Devices

### DISCUSSION THREAD

**Discussion by JoeTheTitan** [PrismLauncher/PrismLauncher#1651] [Ideas]
I think a great feature to add to prism launcher would be to add sync with cloud services, similar to how the mobile emulators Delta and Ignited work. All you do is log in with a Google or Dropbox account and it will automatically sync for you. Maybe you could add the option to only sync certain folders or just sync the entire appdata folder across devices

Thank You.

**Comment by TheKodeToad** [comment]
https://github.com/PrismLauncher/PrismLauncher/issues/891 is this the same sort of thing? if not, please move this discussion to an issue.
